specs/007-uuid-generation/quickstart.md

```
@@ -0,0 +1,121 @@
+- Feature branch: `git checkout 007-uuid-generation`
+
+## Setup
+
+### 1. Add uuid dependency
+
+```bash
+cargo add uuid --features v5
+```
+
+This adds to `Cargo.toml`:
+```toml
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)